Header header
string[] name
geometry_msgs/PoseStamped[] target
geometry_msgs/PoseStamped[] stance
float64[] swing_clearance
